dev/CommandBarFlyout/CommandBarFlyout_themeresources.xaml

```diff
@@ -750,6 +754,8 @@
         <contract7Present:Setter Property="CommandBarOverflowPresenterStyle" Value="{StaticResource CommandBarFlyoutCommandBarOverflowPresenterStyle}" />
         <contract7NotPresent:Setter Property="CommandBarOverflowPresenterStyle" Value="{StaticResource CommandBarFlyoutCommandBarOverflowPresenterStyleRS4}" />
         <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}" />
+        <Setter Property="local:CommandBarFlyoutCommandBarAutomationProperties.ControlType" Value="Menu" />
+        <Setter x:Uid="CommandBarFlyoutCommandBarLocalizedControlType" Property="AutomationProperties.LocalizedControlType" Value="menu" />
```

Comment:
Are we using a setter here so that the compiler is happy?

Reply:
We need the setter so we have something with an x:Uid to set a value on using a localized string.  We give it a value because the XAML compiler will complain otherwise, but its value will be overridden by the value in the localized string `CommandBarFlyoutCommandBarLocalizedControlType.Value`.